ballerina/Package.md

```diff
@@ -0,0 +1,12 @@
+## Package overview
+
+This module provides functions for doing vector operations including calculating l1 and l2 norm, dot product, cosine similarity, euclidean distance and manhattan distance.
```

Comment:
```suggestion
This module provides functions for doing vector operations including calculating the `l1` and `l2` norm, dot product, cosine similarity, euclidean distance, and Manhattan distance.
```

Reply:
Addressed in https://github.com/ballerina-platform/module-ballerina-math.vector/pull/1/commits/1fe0f828ef1ea29a4fc27924af5f541af69729ff.